x-pack/plugins/code/public/components/query_bar/components/options.tsx

```diff
@@ -135,7 +163,8 @@ export class SearchOptions extends Component<Props, State> {
       <div>
         <div className="kuiLocalSearchAssistedInput__assistance">
           <EuiButtonEmpty size="xs" onClick={this.toggleOptionsFlyout}>
-            Options
+            <EuiAvatar color="#70DAD1" size="s" type="space" name={repoScope.length.toString()} />
```

Comment:
Can I ask that we not use Spaces Avatars for indicating amounts? EUI (and Kibana) has a pattern of using the `EuiNotificationBadge` ([bottom of this page](http://localhost:8030/#/display/badge)) for this pattern. This way we will all be consistent when displaying similar patterns.

We use it with filter groups, the global filter bar and facets:


<img width="382" alt="Screen Shot 2019-04-15 at 12 20 00 PM" src="https://user-images.githubusercontent.com/549577/56148881-4849fa00-5f79-11e9-8c9e-cc09aba21e4a.png">
<img width="186" alt="Screen Shot 2019-04-15 at 12 22 40 PM" src="https://user-images.githubusercontent.com/549577/56148883-4849fa00-5f79-11e9-849c-07af7a73544b.png">
<img width="231" alt="Screen Shot 2019-04-15 at 12 19 36 PM" src="https://user-images.githubusercontent.com/549577/56148887-4a13bd80-5f79-11e9-915d-476bb48875af.png">

Reply:
Thanks a lot! didn't know this before